README.md

```diff
@@ -27,9 +27,9 @@ This can be useful in several ways:
 
 Kubernetes: `kubectl get pods`
 
-Argo: Visit https://argocd.idpbuilder.cnoe.io.local:8443/
+Argo: Visit https://argocd.idpbuilder.cnoe.io.localtest.me:8443/
 
-Backstage: http://backstage.idpbuilder.cnoe.io.local/
+Backstage: http://backstage.idpbuilder.cnoe.io.localtest.me/
```

Comment:
I think it needs the port number here to connect to ingress-nginx

```
 http://backstage.idpbuilder.cnoe.io.localtest.me:8443
```

Reply:
Yes as the we expose on the kind cluster and docker the ports 8443 and 8880. Question: Why do you export on kind/docker such ports instead to keep the standards: 80 and 443 ?

Reply:
This is a good question. My guess is that they were using high number ports to avoid seeing to be a privileged user on some systems

Reply:
Such ports should be configurable and defaulted within the kind config file to: 80, 443. I will open a separate issue about that